bolt/test/X86/profile-quality-reporting.test

```diff
@@ -0,0 +1,6 @@
+## Check profile quality stats reporting
+RUN: yaml2obj %p/Inputs/blarge_new.yaml &> %t.exe
+RUN: llvm-bolt %t.exe -o %t.out --pa -p %p/Inputs/blarge_new.preagg.txt | FileCheck %s
+CHECK: among the hottest 5 functions top 5% function CFG discontinuity is 100.00%
+CHECK: among the hottest 5 functions top 5% call graph flow conservation gap is 60.00%
+CHECK: among the hottest 5 functions top 5% CFG flow conservation gap is 45.53% (weighted) and 96.87% (worst)
```

Comment:
@ShatianWang, does it make sense to group stats? I.e. add stats after a header that says `profile quality metrics for hottest/top N functions...`.